lexers/embedded/robot.xml

```diff
@@ -0,0 +1,81 @@
+<lexer>
+  <config>
+    <name>Robot Framework</name>
+    <alias>robot</alias>
+    <filename>*.robot</filename>
+    <mime_type>text/x-robotframework</mime_type>
+  </config>
+  <rules>
+    <state name="root">
+      <rule>
+        <include state="whitespace"/>
+      </rule>
+      <rule pattern="^\*{3}\s*Settings\s*\*{3}">
+        <token type="Keyword"/>
+      </rule>
+      <rule pattern="^\*{3}\s*Test Cases\s*\*{3}">
+        <token type="Keyword"/>
+      </rule>
+      <rule pattern="^\*{3}\s*Variables\s*\*{3}">
+        <token type="Keyword"/>
+      </rule>
+      <rule pattern="^\*{3}\s*Keywords\s*\*{3}">
+        <token type="Keyword"/>
+      </rule>
+      <rule pattern="\[Documentation\]|\[Tags\]|\[Setup\]|\[Teardown\]|\[Template\]|\[Timeout\]">
```

Comment:
Right now Anvil doesn't highlight tags of type 'NameTag'. We could change it to 'Name', but the default theme in Anvil colors 'Name' the same color as the regular text, but that can be changed by creating a custom style.js file

We could also change this to Keyword which will definitely get highlighted... what do you think?

Reply:
I see. I think 'Keyword' should be unique in order to be easily distinguishable. We can use simply 'Name'

Reply:
Sure, let's have it as 'Name'.